packages/react-scripts/config/paths.js

```diff
@@ -56,6 +57,13 @@ const resolveModule = (resolveFn, filePath) => {
   return resolveFn(`${filePath}.js`);
 };
 
+let cachePath = resolveApp('node_modules/.cache');
+try {
+  fs.accessSync(cachePath, fs.constants.W_OK | fs.constants.R_OK);
```

Comment:
We shouldn't fall back to an OS temp directory if `node_modules/.cache` doesn't exist. This would certainly be the case for a fresh project. In my testing of setting `cacheLocation`, the folder will be created for you during `npm start` or `npm run build`.

If we can find a case where it doesn't work without the directory being present, we should instead try to create the directory, but my preference would be to not handle this case unless we have a valid reproduction.

Reply:
I am sure that eslint will create directory of `cacheLocation` if it doesn't exist by `mkdirp`. [debug these code](https://github.com/eslint/eslint/blob/110cf962d05625a8a1bf7b5f4ec2194db150eb32/lib/cli-engine/cli-engine.js#L832-L835).